app/dao/service_sms_sender_dao.py

```diff
@@ -0,0 +1,18 @@
+from app import db
+from app.models import ServiceSmsSender
+
+
+def update_service_sms_sender(service, sms_sender):
```

Comment:
Do we need transactional here?

Reply:
That's what I wanted to chat about. The update and create service is wrapped in transactional which is why I left it out here. Not sure if this will be called without update but it might. Can they both have transactional?

Reply:
Yeah I think it's okay, it's just if we use it elsewhere that isn't wrapped in transactional. E.g. When we do this in `update_service` on API?